helpers/postgres.rb

```diff
@@ -127,7 +127,7 @@ def generate_postgres_options(flavor: nil, location: nil)
 
     options.add_option(name: "family", values: Option::POSTGRES_FAMILY_OPTIONS.keys, parent: "location") do |flavor, location, family|
       if location.aws?
-        ["m8gd", "i8g"].include?(family) || (Option::AWS_FAMILY_OPTIONS.include?(family) && @project.send(:"get_ff_enable_#{family}"))
+        ["m8gd", "i8g", "i8ge", "i7i", "i7ie"].include?(family) || (Option::AWS_FAMILY_OPTIONS.include?(family) && @project.send(:"get_ff_enable_#{family}"))
```

Comment:
The left part of the `||` condition are for families that don't require feature flag. I think we would want these to be behind feature flag.